bot/cogs/owner_cog.py

```diff
@@ -165,13 +165,13 @@ async def get(self, ctx, lines: int):
     @eval_bot.command()
     @commands.is_owner()
     async def bot(self, ctx, *, code):
-        code = code.strip('`')
-        res = eval(code)
-        if inspect.isawaitable(res):
-            res = await res
-
-        
-        await ctx.send(res)
+        code = code.replace('```python', '')
+        code = code.replace('`', '')
+        code = code.replace('\n', '\n\t')
+        t=[None]
+        code = 'async def foobar():\n\t' + code + '\nt[0] = loop.create_task(foobar())'
+        exec(code, {'asyncio': asyncio, 'bot': self.bot, 'code': code, 'ctx': ctx, 'loop': asyncio.get_running_loop(), 'self': self, 't': t})
```

Comment:
lets pull this dictionary out into its own variable and also we dont need to pass in self here

Reply:
Dome